internal/pkg/agent/cmd/logs_test.go

```diff
@@ -99,7 +99,7 @@ func TestGetLogFilenames(t *testing.T) {
 
 	t.Run("does not return non-log entries", func(t *testing.T) {
 		dir := t.TempDir()
-		createFile(t, dir, "excluded")
+		createFileEmpty(t, dir, "excluded")
```

Comment:
Just checking: has this been renamed because of name collision within the same package?